keras/losses.py

```diff
@@ -372,7 +372,9 @@ def __init__(self,
 class MeanAbsolutePercentageError(LossFunctionWrapper):
   """Computes the mean absolute percentage error between `y_true` and `y_pred`.
 
-  `loss = 100 * abs(y_true - y_pred) / y_true`
+  `loss = 100 * abs((y_true - y_pred) / y_true)`
+  
+  Note that to avoid division by zero error, a small value epsilon is added to the denominator.
```

Comment:
"Note that to avoid dividing by zero, a small epsilon value is added to the denominator."

Reply:
@fchollet, I have modified as per your advise, Please take a look on it.